### ISSUE TITLE
Bug on windows with make setup

### ISSUE THREAD

**Issue by melonora** [linkml/linkml#1750]
**Describe the bug**
The make setup fails due to incorrect syntax when creating directories with `mkdir -p`.

**To reproduce**
On windows create a project using the linkml cookiecutter project and then run the `make setup` command. An error
is returned when running line 151 of the `Makefile`, `mkdir -p`. This is because the path contains 2 forward slashes making the syntax incorrect.

**Expected behavior**
To have the setup complete succesfully

**Screenshots**
![image](https://github.com/linkml/linkml/assets/22346363/7c8ca5e0-2de6-482f-9bbb-c46a4bb8e770)

**About your computer (if applicable, please complete the following information):**
 - OS: Windows

**Comment by dalito** [comment]
As a workaround you may run the command in the bash shell coming with git for Windows.

**Comment by melonora** [comment]
I can have a look for sure:)

**Comment by nlharris** [comment]
There was a fix for another Windows-only issue recently: https://github.com/linkml/linkml/issues/2270
Is it possible that helps here?

**Comment by dalito** [comment]
@nlharris - Unfortunately not. The problem is that some make command execute unix shell commands which don't work in cmd or powershell. But they work in the bash shell coming with git as I proposed above.

@melonora - Could this issue be closed? I think it is not really a fixable bug but a known limitation of the current tools.

On Windows `make` is quite inconvenient. I recently tried using [just](https://just.systems/man/en/) as command runner which looks like a nice alternative. However, it requires that the user installs yet another tool.

Providing support for a better cross-platform command runner may be worth a new issue/feature request if there is interest.
